src/app.js

```diff
@@ -10,12 +10,13 @@ const loadConfig = require('./lib/config')
 const RoutingTables = require('./lib/routing-tables')
 const RouteBuilder = require('./lib/route-builder')
 const RouteBroadcaster = require('./lib/route-broadcaster')
+const CurveCache = require('./lib/curve-cache')
```

Comment:
@emschwartz I think maybe we should discuss this in the first meeting of the routing working group? The data structures ilp-kit uses are an implementation detail, but if curves from quotes are kept longer/shorter than curves from routes, that does change the functionality. It feels to me like we haven't clearly defined the functional requirements of the connector, with respect to e.g. reacting to price changes. For instance, is the `expiresAt` field of a quote response an indication only, or should it always be honored even if market prices changed?